### ISSUE TITLE
IN_BAILIWICK_ADDR_MISMATCH should give more information

### ISSUE THREAD

**Issue by matsduf** [zonemaster/zonemaster-engine#467]
Test Case [Consistency05] (as currently in the develop branch) returns the message IN_BAILIWICK_ADDR_MISMATCH when there is a mismatch of the IP address for an in-bailiwick name server between delegation and authoritative zone. It then returns the names and IP addresses of the name servers as stated in the delegation but information on what has been found in the authorititative zone is not listed, i.e. it is impossible to see what the problem is.
```
$ zonemaster-cli --ns a.nic.fi/46.21.96.58 --ns a.nic.fi/2a02:750:12::53 --ns b.nic.fi/212.85.74.18 --ns b.nic.fi/2001:470:28:5a0::53 fi  --ds "2701,13,2,f93108ff18ef62aa0c5ba9696e259f06cf971a016763191658376be947fee126"  --test CONSISTENCY/consistency05 --raw
   3.05 ERROR     CONSISTENCY:IN_BAILIWICK_ADDR_MISMATCH addresses=a.nic.fi./2a02:750:12::53;a.nic.fi./46.21.96.58;b.nic.fi./2001:470:28:5a0::53;b.nic.fi./212.85.74.18
```

[Consistency05]: https://github.com/zonemaster/zonemaster/blob/develop/docs/specifications/tests/Consistency-TP/consistency05.md

**Comment by vlevigneron** [comment]
@matsduf I just started to look at that. I noticed (but perhaps is is what we want), that there is no messages specified for IN_BAILIWICK_ADDR_MISMATCH and OUT_OF_BAILIWICK_ADDR_MISMATCH cases. Is it something we decided in a past discussion ?

**Comment by vlevigneron** [comment]
In this specific case (aka IN_BAILIWICK_ADDR_MISMATCH), it lists what is found in parent zone AND is missing from child.
Perhaps we could add, as an information, without change the current messages and there content, list of servers/IPs found in delegation AND list of servers/IPs found in authoritative zone.

**Comment by matsduf** [comment]
No, there should be messages for IN_BAILIWICK_ADDR_MISMATCH and OUT_OF_BAILIWICK_ADDR_MISMATCH, respectively. What you want in "mismatch" is something like "found in parent, found in zone".

**Comment by vlevigneron** [comment]
OK, I can do that.

**Comment by vlevigneron** [comment]
Perhaps it needs to be refined, but tell me if the corresponding PR is more or less what you have in mind.

**Comment by matsduf** [comment]
Fixed by #515.
